packages/core/lib/generators/integrations/image_integration.dart

```diff
@@ -32,6 +36,9 @@ class ImageIntegration extends Integration {
     this._assetName, {
     this.size,
     this.flavors = const {},
+    this.isAnimation = false,
+    this.duration = Duration.zero,
+    this.frames = 1,
```

Comment:
Can we export `ImageAnimation` rather than splitting fields into the instance? Which one looks better?

Reply:
Moved all fields to `AssetGenImageAnimation`.